team/tech/coordination.md

```diff
@@ -2,6 +2,22 @@
 
 As a totally distributed team, it is important to have team practices that give one another insight into what we're working on, identify opportunities to collaborate, and un-block one another. This page describes some of the practices that we adopt towards these goals.
 
+
+(coordination:team-syncs)=
+## Weekly team syncs
+
+Every Monday, the 2i2c Hub Operations team conducts a team sync to get on the same page and coordinate their work for the week.
+
+1. **Create an issue using [the team sync issue template](https://github.com/2i2c-org/team-compass/issues/new?assignees=&labels=team-sync&template=team-sync.md)**. This issue is our space to discuss, update, and sync.
+2. **Team members give their responses**. You can copy/paste the template, and then give your responses in comments to the issue. You shouldn't feel forced to add content if you can't think of anything, use it as much as is useful.
```

Comment:
> You shouldn't feel forced to add content if you can't think of anything, use it as much as is useful.

That's super important! People should not feel coerced to fill the blanks. With time, it should be natural and stress-free to share stuff here...